system-tests/projects/angular-signals/src/signals-required/signals-required.component.cy.ts

```diff
@@ -0,0 +1,150 @@
+import { signal, computed } from '@angular/core'
+import { SignalsRequiredComponent } from './signals-required.component'
+import { createOutputSpy } from 'cypress/angular-signals'
+
+// NOTE: if this is the only test in your test suite, this error will continually throw until the fixture is closed.
+it('errors on required props missing', (done) => {
+  cy.once('uncaught:exception', (e) => {
+    expect(e.message).to.include('Input is required but no value is available yet.')
+    done()
+
+    return true
+  })
+
+  cy.mount(SignalsRequiredComponent)
+})
+
+it('can handle input signal as primitive value (title prop)', () => {
+  cy.mount(SignalsRequiredComponent, {
+    componentProperties: {
+      title: 'Signals Component as Primitive',
+      count: signal(0),
+    },
+  })
+
+  cy.get('[data-cy="signals-required-component-title-display"]').should('contain.text', 'Signals Component as Primitive')
+})
+
+// FIXME: we currently need to allow this as there isn't a great way to set input signals in a component due to how input signals are instantiated
```

Comment:
Do we have an issue logged for this work?

Reply:
logged https://github.com/cypress-io/cypress/issues/29732

Reply:
added a comment too in 001813ebf9